views/layouts/admin_sidebar.php

```diff
@@ -0,0 +1,24 @@
+<form id="admin-search-form" class="text-center" action="#">
+
+  <div class="form-group">
+    <label for="title">Title</label>
+    <input type="text" class="form-control" id="title" placeholder="Title">
+  </div>
+
+  <div class="form-group">
+    <label for="author">Authors:</label>
+    <select class="form-control" id="author" >
+      <option>Select Author</option>
+    </select>
+  </div>
+
+  <div class="form-group">
+    <label for="price">Price</label>
+    <input type="number" class="form-control" id="price_from" placeholder="from" min="0">
+    <input type="number" class="form-control" id="price_to" placeholder="to" min="0">
```

Comment:
"id" and "class" attributes should use dashes instead of underscore. Underscore is not supported by all browsers correctly.
In this case: price_from / price_to need to be updated to price-from and price-to
Every input field should have a name. It's the attribute that is passed in $_POST on submit.